pkg/gather/clusterconfig/constants.go

```diff
@@ -0,0 +1,11 @@
+package clusterconfig
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	datahubGroupVersionResource = schema.GroupVersionResource{
+		Group: "installers.datahub.sap.com", Version: "v1alpha1", Resource: "datahubs",
+	}
+)
```

Comment:
I think we already have `const.go` after last Marcell commit. Please put it there.